server.py

```diff
@@ -235,6 +235,7 @@ def do_get_response(mongo_id):
 
 @app.route('/queue', methods=['POST'])
 def do_queue():
+    logger.info("Begin Queueing")
```

Comment:
Needs to be logging the `tx_id` - don't think it's bound to the logger instance you're using here